src/transactions/typeTag/index.ts

```diff
@@ -338,25 +338,25 @@ export class TypeTagStruct extends TypeTag {
 export class StructTag extends Serializable {
   public readonly address: AccountAddress;
 
-  public readonly module_name: Identifier;
+  public readonly moduleName: Identifier;
 
   public readonly name: Identifier;
 
-  public readonly type_args: Array<TypeTag>;
+  public readonly typeArgs: Array<TypeTag>;
```

Comment:
depends if this is deserialized from JSON / it's used directly there.  If it's not then we can change it.

Do we use BCS for all of our communication to the node?